gobblin-utility/src/main/java/gobblin/util/SchedulerUtils.java

```diff
@@ -97,9 +107,7 @@ public boolean accept(File file, String name) {
           rootPropsCopy.putAll(rootProps);
           loadJobConfigsRecursive(jobConfigs, rootPropsCopy, jobConfigFileExtensions, file);
         } else {
-          int pos = file.getName().lastIndexOf(".");
-          String fileExtension = pos >= 0 ? file.getName().substring(pos + 1) : "";
-          if (!jobConfigFileExtensions.contains(fileExtension)) {
+          if (!jobConfigFileExtensions.contains(Files.getFileExtension(file.getName()).toLowerCase())) {
             // Not a job configuration file, ignore.
```

Comment:
Can we add a LOG.warn() in order to document anytime we skip over a config file because it doesn't have a proper extension